src/Earnings/SFA.DAS.Earnings/Application/Earnings/PriceEpisodePeriodisedValuesBuilder.cs

```diff
@@ -98,16 +98,23 @@ public static PriceEpisodePeriodisedValues BuildCoInvestmentValues(JoinedPriceEp
         };
     }
     
-    public static PriceEpisodePeriodisedValues BuildNthIncentivePaymentValues(JoinedPriceEpisode joinedPriceEpisode, short academicYear, string attributeName, string additionalPaymentType, int n)
+    public static PriceEpisodePeriodisedValues BuildNthIncentivePaymentValues(JoinedEarningsApprenticeship joinedApprenticeship, JoinedPriceEpisode joinedPriceEpisode, short academicYear, string attributeName, string additionalPaymentType, int n)
     {
-        var additionalPayments = GetAdditionalPayments(joinedPriceEpisode, additionalPaymentType);
```

Comment:
Do we need this private method following this refactor? Assume these production code changes were to fix an issue uncovered by the new acceptance tests?